ml/pre-processing/PreProcessingConfig.scala

```diff
@@ -1,9 +1,9 @@
-object PreProcessingConfig {
-
-  val keyspacekeyspace = "gsoc3"
+object PreProcessingConfig extends Serializable{
```

Comment:
I would recommend converting your python config to JSON. That way you can have one config for the entire system.